bangazon/urls.py

```diff
@@ -51,5 +50,10 @@
         Products.as_view({"post": "rate_product"}),
         name="rate-product",
     ),
-    path("products/liked/", Products.as_view({"get": "liked"}), name="liked-products")
+    path("products/liked/", Products.as_view({"get": "liked"}), name="liked-products"),
+    path(
```

Comment:
I think I misled the team about using the url paths. If you setup an @action correctly you don't need to set up a url path. All that to say you can remove lines 54-58. I'll take out line 53 that was mine in my next pull request.